example_bytecode_test.go

```diff
@@ -1,4 +1,4 @@
-package duktape_test
+package duktape
```

Comment:
Another opportunity is to move example to README.md. As I can see there is already couple of examples thou.